algorithmic_efficiency/workloads/librispeech_deepspeech/librispeech_jax/workload.py

```diff
@@ -0,0 +1,50 @@
+import functools
+import itertools
+import math
+from typing import Dict, Tuple
+
+from absl import flags
+from absl import logging
+from flax import jax_utils
+import flax.linen as nn
+import jax
+from jax import lax
+import jax.numpy as jnp
+import numpy as np
+import optax
+
+from algorithmic_efficiency import spec
+from algorithmic_efficiency.workloads.librispeech_conformer.librispeech_jax.workload import \
+    LibriSpeechConformerWorkload
```

Comment:
do we need to check if model_state is not none here? https://github.com/mlcommons/algorithmic-efficiency/blob/main/algorithmic_efficiency/workloads/librispeech_conformer/librispeech_jax/workload.py#L61

also do we need to return model_state here instead of None? https://github.com/mlcommons/algorithmic-efficiency/blob/main/algorithmic_efficiency/workloads/librispeech_conformer/librispeech_jax/workload.py#L82

Reply:
1) it will fail automatically if batch_stats is not passed in 
2) we return model_state as None for eval mode and we ignore that returned model_state in eval code path so it's fine